pyanaconda/core/constants.py

```diff
@@ -59,6 +59,8 @@
 IMAGE_DIR = MOUNT_DIR + "/image"
 INSTALL_TREE = MOUNT_DIR + "/source"
 BASE_REPO_NAME = "anaconda"
+ANACONDA_RUN_DIR = "/run/anaconda"
+ANACONDA_BUS_ADDR_FILE = "bus.address"
```

Comment:
I think that there should be a full path to the file. It is not very useful this way and you could remove `_make_bus_address_file_path`.

Reply:
OK

Reply:
Fixed.